tests/regression.sh

```diff
@@ -37,16 +46,20 @@ function cmp_output()
     printf "$cmd"
 
     if [[ $TRAVIS_OS_NAME == windows ]]; then
-        cmp_tool="diff --strip-trailing-cr -q"
+        cmp_tool="diff --strip-trailing-crq"
     else
-        cmp_tool="cmp -s"  # --silence does not work on Alpine
+        cmp_tool="cmp"
     fi
 
-    if eval $cmd 2>/dev/null | eval $cmp_tool $expected_out - > /dev/null; then
+    if eval $cmd 2>cmd.err | tee actual.out | eval $cmp_tool $expected_out - >cmp_tool.out; then
```

Comment:
I'd recommend using `mktemp -d` to create a temporary dir (by default in /tmp) and put all temporary files there, cleanup can be omitted.